src/SFA.DAS.Commitments.Api.UnitTests/Mapping/Service/Facets/WhenExtractingApprenticeshipStatuses.cs

```diff
@@ -64,7 +64,7 @@ public void ShouldOnlyCreateOneFacetPerStatus()
         public void ShouldHaveWaitingForStopped()
         {
             _data.Add(new Apprenticeship { PaymentStatus = PaymentStatus.Withdrawn });
-            _data.Add(new Apprenticeship { FirstName = "Started", PaymentStatus = PaymentStatus.Active, StartDate = DateTime.Now.AddMonths(-1) });
+            _data.Add(new Apprenticeship { FirstName = "Started", PaymentStatus = PaymentStatus.Active, StartDate = _currentDateTime.Object.Now.AddMonths(-1) });
```

Comment:
Unrelated to story - unit test was sensitive to the date it was run and had broken, so fixed it.